examples/solids/Makefile

```diff
@@ -34,6 +34,16 @@ LDFLAGS = $(call pkgconf, --libs-only-L --libs-only-other $(PETSc.pc) $(ceed.pc)
 LDFLAGS += $(patsubst -L%, $(call pkgconf, --variable=ldflag_rpath $(PETSc.pc))%, $(call pkgconf, --libs-only-L $(PETSc.pc) $(ceed.pc)))
 LDLIBS = $(call pkgconf, --libs-only-l $(PETSc.pc) $(ceed.pc)) -lm
 
+# Enzyme-AD
+ENZYME_LIB ?=
+ENZYMEFLAGS = -Xclang -load -Xclang $(ENZYME_LIB) -O2 -fno-unroll-loops -mllvm -enzyme-print
+
+ifneq ($(ENZYME_LIB),)
+  CFLAGS += $(ENZYMEFLAGS)
+else
+  $(error Must export ENZYME_LIB=/path/to/ClangEnzyme-XX.so)
+endif
```

Comment:
Before merging, we should make this an optional dependency and make it a run-time error if the user tries to use a problem that uses Enzyme without having Enzyme installed.

Reply:
Right, I should make that optional but since `ENZYME_LIB` is needed to compile the code, I think we should export it before the compilation.